tests/lib/utils/import-test.js

```diff
@@ -21,3 +21,26 @@ describe('getSourceModuleName', () => {
     expect(importUtils.getSourceModuleName(node)).toStrictEqual('Model');
   });
 });
+
+describe('getImportIdentifiers', () => {
```

Comment:
Can you add a test case for a default import?

Reply:
👍